src/components/LanguagePicker/MenuItem.tsx

```diff
@@ -48,86 +42,58 @@ const MenuItem = forwardRef(({ displayInfo, ...props }: ItemProps, ref) => {
   const { progress, words } = getProgressInfo(approvalProgress, wordsApproved)
 
   return (
-    <ChakraMenuItem
-      as={BaseLink}
-      ref={ref}
-      flexDir="column"
-      w="full"
-      mb="1"
-      display="block"
-      pt="2 !important"
-      alignItems="start"
-      borderRadius="base"
-      bg={isCurrent ? "background.base" : "transparent"}
-      color="body.base"
-      textDecoration="none"
-      onFocus={(e) => {
-        e.target.scrollIntoView({ block: "nearest" })
-      }}
-      scrollMarginY="8"
-      _hover={{
-        bg: "primary.lowContrast",
-        textDecoration: "none",
-        "p.language-name": { color: "primary.base" },
-      }}
-      _focus={{ bg: "primary.lowContrast" }}
-      sx={{
-        p: {
-          textDecoration: "none",
-          overflow: "hidden",
-          textOverflow: "ellipsis",
-          whiteSpace: "nowrap",
-        },
-      }}
-      href={asPath}
-      locale={localeOption}
+    <CommandItem
+      value={localeOption}
+      className={cn(
+        "group mb-1 flex-col items-start rounded pt-2 text-body hover:bg-primary-low-contrast",
+        isCurrent
+          ? "bg-background hover:bg-primary-low-contrast"
+          : "bg-transparent"
+      )}
       {...props}
+      // TODO double check this
+      // sx={{
+      //   p: {
+      //     textDecoration: "none",
+      //     overflow: "hidden",
+      //     textOverflow: "ellipsis",
+      //     whiteSpace: "nowrap",
+      //   },
+      // }}
```

Comment:
Haven't encountered any need for this while testing it out.

Reply:
Thank, yes, I was trying to see where this was coming from but I think we could clean this up since it doesn't seem to add anything atm.